packages/console/src/components/Sidebar/index.tsx

```diff
@@ -1,9 +1,38 @@
 import React from 'react';
+import { useTranslation } from 'react-i18next';
 
+import Item from './components/Item';
+import Section from './components/Section';
+import { sections } from './consts';
+import Gear from './icons/Gear';
 import * as styles from './index.module.scss';
 
 const Sidebar = () => {
-  return <div className={styles.sidebar}>Sidebar</div>;
+  const { t: tSection } = useTranslation(undefined, {
```

Comment:
Not using namespaces?

Reply:
didn't follow this, mind elaborating?

Reply:
OK, in other words, why the first param is undefined? The i18next documentation says this param is `namespace`, and it seems to be useful?

<img width="765" alt="image" src="https://user-images.githubusercontent.com/12833674/156191523-e1461efe-a346-4323-9a54-6460b562ebf0.png">

<img width="747" alt="image" src="https://user-images.githubusercontent.com/12833674/156191452-b739328a-febf-4da2-b4eb-0e470352678c.png">

Reply:
would prefer to fall back to the default namespace instead of explicitly passing, since the default `useTranslation()` uses `undefined` under the hood

Reply:
ok